benchmark/compute_metrics.py

```diff
@@ -293,11 +293,11 @@ def read_out(fn):
         for line in f.readlines():
             row = json.loads(line)
             txyz.append([row["time"], row["position"]["x"], row["position"]["y"], row["position"]["z"]])
-            stat.append(row.get("stationary", False))
-            if (row.get("mean")):
-                bga.append(to_arr(row["mean"]["bga"]))
-                baa.append(to_arr(row["mean"]["baa"]))
-                bat.append(to_arr(row["mean"]["bat"]))
+            stat.append(row["stationary"])
+            if (row["biasMean"]):
```

Comment:
I think these will crash if the field in question is missing. That's why .get() was used instead of direct [] access

Reply:
Good catch. I didn't intend to change the accessing, I think I used an old branch as base by accident. Fixed now.